### PR TITLE
Update packages for docs

### PR DESCRIPTION
**Pull request by navidcy**
"Live on the bleeding edge."

### CODE REVIEW

File: docs/Manifest.toml [hunk 1]
```diff
@@ -153,15 +153,9 @@ uuid = "2e619515-83b5-522b-bb60-26c02a35a201"
...
+git-tree-sha1 = "b7e3d17636b348f005f11040025ae8c6f645fe92"
 uuid = "e2ba6199-217a-4e67-a87a-7c52f15ade04"
-version = "0.1.3"
-
-[[EzXML]]
-deps = ["Printf", "XML2_jll"]
-git-tree-sha1 = "0fa3b52a04a4e210aeb1626def9c90df3ae65268"
-uuid = "8f5d6c58-4d21-5cfd-889c-e3ad7ee6a615"
-version = "1.1.0"
+version = "0.1.6"
```
Comment (francispoulin) [inline docs/Manifest.toml:158]: Are we going from 1.1.0 to 0.1.6?

**Review by francispoulin — APPROVED** [review]
One strange version jump seems to be backwards but I suspect it's because I misunderstood something.

File: docs/Manifest.toml (hunk 1 above)
Comment (navidcy) [reply docs/Manifest.toml:158]: I’ll look into this.

Comment (francispoulin) [reply docs/Manifest.toml:158]: Thanks

Comment (navidcy) [inline docs/Manifest.toml:158]: @francispoulin you misunderstood :)
the code here implies remove EzXML completely and upgrade ExprTools 0.1.3->0.1.6

Comment (navidcy) [reply docs/Manifest.toml:158]: @francispoulin you misunderstood :)
the code here implies remove EzXML completely and upgrade ExprTools 0.1.3->0.1.6

**Comment by navidcy** [comment]
> One strange version jump seems to be backwards but I suspect it's because I misunderstood something.

@francispoulin you misunderstood :)
the manifest here implies that EzXML is removed completely and ExprTools upgraded from 0.1.3 -> 0.1.6

**Comment by francispoulin** [comment]
Thanks!

Francis


________________________________

**Pull request merged**